### PR TITLE
UD-279-ds-match-read-mentee-mentor-endpoint-to-read-feedback

### PR DESCRIPTION
**Pull request by gpwolfe**
## Description
mentor_router.py and mentee_router.py used a different read query format than feedback_router.py.
Queries for read/mentee and read/mentor accepted an `Optional[Dict]` as input, while read/feedback used a class included in schema.py to define acceptable query fields.
Changes include:
- adding MenteeOptions and MentorOptions classes (to match existing FeedbackOptions) to schema
- using the new classes in the respective _router.py files at the read endpoints
- removing now-unused imports from router files

## Jira Ticket
[Jira Ticket UD-279](https://bloomtechlabs.atlassian.net/jira/software/c/projects/UD/boards/17?modal=detail&selectedIssue=UD-279)
## Type of Change
- [X] Bug fix

## Checklist
- [x] My code follows PEP8 style guide
- [x] I have removed unnecessary print statements from my code
- [x] I have made corresponding changes to the documentation if necessary
- [x] My changes generate no errors
- [x] No commented-out code
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

## Loom Video
[Loom Video](https://www.loom.com/share/cf43722554ef4437981ed0601f0055f9)

### CODE REVIEW

**Review by FatimahSarwar — APPROVED** [review]
Awesome job updating the schema.

**Review by jtorres03 — APPROVED** [review]
Great job! Changes are well organized and formatted. Thanks for putting in the work to make the code a bit more uniform.

**Review by jinjahninjah — APPROVED** [review]
Great work! Thank you for taking the time to refactor the code to be more uniform stylistically. 👍 :shipit:

File: app/schema.py [hunk 1]
```diff
@@ -52,6 +52,28 @@ class MentorUpdate(ExtraForbid):
     accepting_new_mentees: Optional[bool]
 
 
+class MentorOptions(ExtraForbid):
```
Comment (SJRUK2000) [inline app/schema.py:55]: Greatly appreciate the buildout on this as we are now a step closer to being able to work with Mentor Data!

File: app/schema.py [hunk 2]
```diff
@@ -95,6 +117,27 @@ class MenteeUpdate(ExtraForbid):
     in_project_underdog: Optional[bool]
 
 
+class MenteeOptions(ExtraForbid):
```
Comment (SJRUK2000) [inline app/schema.py:120]: Thank you for completing this, it brings us closer to being able to work with mentee data.

**Review by SJRUK2000 — APPROVED** [review]

**Pull request merged**